packages/core/src/api/exporters/html/util/serializeBlocksExternalHTML.ts

```diff
@@ -144,6 +144,15 @@ function serializeBlock<
     // We wrap the output in an `li` element for list items, and so we want to
     // add the attributes to that element instead as it is the "root".
     if (!listType) {
+      // TODO: This is specifically for default blocks, as default props get
```

Comment:
can you explain more what the code does? You're now explaining "why" it's here, but what does it do? why do we move styles from ret.dom to ret.dom.firstchild?